x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/signals_mapping.json

```diff
@@ -391,6 +391,9 @@
           },
           "threshold_result": {
             "properties": {
+              "from": {
+                "type": "date"
```

Comment:
The concept of storing the actual time range that was searched would be useful for all rule types, possibly as an intermediate step if we don't want to/can't store the entire actual query that the detection engine used (which would include exceptions, time ranges, etc). Nothing to change in this PR, just a thought for future work.